### ISSUE TITLE
Jaccard loss

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#341]
Another useful loss function for semantic segmentation tasks. It was available in [keras-contrib](https://github.com/keras-team/keras-contrib), [losses/jaccard.py](https://github.com/keras-team/keras-contrib/tree/master/keras_contrib/losses)

**Comment by qlzh727** [comment]
Moved back to keras-cv. I think we would like this to be more cv specific, eg for bounding box etc. We will keep this in keras-cv for now, until we feel there are generic use case support in core keras.

**Comment by qlzh727** [comment]
Feel free to send PR to keras-cv/losses.

**Comment by LukeWood** [comment]
The KerasCV team is attempting to narrow the package scope in order to more efficiently use our limited resources. As such, we are only accepting contributions directly related to items on the 6 month roadmap for the package.

If you feel this feature could be particularly useful, or would like to contribute it - please feel free to re-open this issue and ping LukeWood, ianstenbit or tanzhenyu

Feel free to read more on the roadmap:
https://github.com/keras-team/keras-cv/blob/master/.github/ROADMAP.md
